### ISSUE TITLE
Failed to load class "org.slf4j.impl.StaticLoggerBinder".

### ISSUE THREAD

**Issue by mibby** [Shynixn/BlockBall#34]
blockball-4.1.0-20170804.104410-2-jar-with-dependencies.jar
Paper dev 1178 (Spigot 1.12.1)
```
[23:39:28] [Server thread/INFO]: [BlockBall] Enabling BlockBall v4.1.0-SNAPSHOT
[23:39:28] [Server thread/INFO]: [BlockBall] Loading BlockBall ...
[23:39:28] [Server thread/WARN]: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[23:39:28] [Server thread/WARN]: SLF4J: Defaulting to no-operation (NOP) logger implementation
[23:39:28] [Server thread/WARN]: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[23:39:28] [Server thread/INFO]: Connected to jdbc:sqlite:/server3/plugins/BlockBall/BlockBall.db
[23:39:28] [Server thread/INFO]: [BlockBall] found dependency [WorldGuard] 5.
[23:39:28] [Server thread/INFO]: [BlockBall] failed to hook into [WorldGuard] 5.
[23:39:28] [Server thread/INFO]: [BlockBall] found dependency [WorldGuard] 6.
[23:39:28] [Server thread/INFO]: [BlockBall] hooked successfully into [WorldGuard] 6.
[23:39:28] [Server thread/INFO]: [BlockBall] found dependency [Vault].
[23:39:28] [Server thread/INFO]: [BlockBall] hooked successfully into [Vault].
[23:39:28] [Server thread/INFO]: [BlockBall] found dependency [PlaceholderAPI].
[23:39:28] [Server thread/INFO]: [BlockBall] hooked successfully into [PlaceholderAPI].
[23:39:28] [Server thread/INFO]: [BlockBall] Failed to register the last dependency.
[23:39:28] [Server thread/INFO]: [BlockBall] Enabled BlockBall 4.1.0-SNAPSHOT by Shynixn
```

**Comment by Shynixn** [comment]
Will be fixed update.

**Comment by Shynixn** [comment]
Fixed in latest snapshot blockball-4.1.0-20170812.130734-3-jar-with-dependencies.jar
